public/server/db/Ijgb.json

```
@@ -0,0 +1,3 @@
+{
+  "definition": "I just got back"
```

This should be written as "I Just Got Back" with the first letter of each word in caps like the README says.
```suggestion
  "definition": "I Just Got Back"
```